### DISCUSSION TITLE
create a PPT template with 2 pics and a headline

### DISCUSSION THREAD

**Discussion by Tasselmi** [marp-team/marp#355] [Q&A] [answered]
Hi,

Firstly thank you guys and your team make such an excellent writing tool for us ！！！

I would also like to use it for professional business presentations, it must use a specific layout for this.

I already have some customization, but get stuck on define a template header. 

An example is shown here:
![shop](https://user-images.githubusercontent.com/12438278/191969138-5ff6f6b8-e515-4bff-bab6-0d200c1d05a2.png)

the header consists of 2 parts：
1. the left part is a logo picture
2. the right part contains of 2 parts too：
2.1 the right top is headline of each PPT page, which writer can write freely
2.2 the right bottom is a picture that likes a long straight line

Can anyone help me？ Thanks. I have got stuck here for 3 days and not solve the problem. 
much appreciate.

**Comment by yhatt** [comment, marked as answer]
This is a simple example to reproduce the layout you were provided.

```html
<!-- header: Header content -->

<style>
  header {
    /* Unset already defined styles in built-in theme */
    all: unset;

    /* Set the position of header */
    position: absolute;
    left: 1rem;
    right: 1rem;
    top: 1rem;

    /* Align text to vertically center */
    display: flex;
    align-items: center;

    /* Sizing */
    box-sizing: border-box;
    min-height: 2em;

    /* Text style */
    font-weight: bold;

    /* Add a border to the bottom */
    border-bottom: 2px solid #ee4d2d;

    /* Make a space to the left for the icon */
    margin-left: 2.5em;
  }

  header::before {
    /* Make an empty content */
    content: '';
    display: block;

    /* Positioning (relative from the header element) */
    position: absolute;
    top: 0;
    left: -2.5em;

    /* Sizing */
    width: 2em;
    height: 2em;

    /* Set icon image as a background */
    background-image: url('https://icongr.am/octicons/markdown.svg?size=128&color=ee4d2d');
    background-repeat: no-repeat;
    background-size: contain;
  }
</style>
```
